src/pages/Profile/Profile.tsx

```diff
@@ -208,9 +208,9 @@ const Profile = () => {
                 </ div>
               }
             </div>
-            <div id="buttonsContainer">
-              <Button label="Save changes" variant="contained" color="secondary" onClick={() => { setEditProfile(false); setChangesMade(false) }} disabled={!changesMade} />
-              <Button label="Cancel" variant="contained" id="grayButton" onClick={() => { setEditProfile(false); setChangesMade(false) }} />
+            <div className="profileButtonsContainer">
+              <Button label="Save changes" variant="contained" color="secondary" onClick={() => { setEditProfile(false); setChangesMade(false) }} className={!changesMade ? 'disabledButton' : 'enabledButton'} />
+              <Button label="Cancel" color="primary" variant="outlined" onClick={() => { setEditProfile(false); setChangesMade(false) }} className={!changesMade ? 'disabledButton' : 'enabledButton'} />
```

Comment:
`className={!changesMade ? 'disabledButton' : 'enabledButton'}` doesn't look right. We should use the default `disabled` prop.

If you want to change the default component, you can change it here - https://github.com/Taraxa-project/taraxa-ui/blob/main/src/components/theme.js#L69